### ISSUE TITLE
@glue42/react-workspaces not found

### ISSUE THREAD

**Issue by mingfang** [Glue42/core#188]
I'm trying to create a Workspace app using the instructions here.
https://www.npmjs.com/package/@glue42/workspaces-ui-react

The install step fails
```
npm install --save @glue42/react-workspaces

npm ERR! 404 Not Found - GET https://registry.npmjs.org/@glue42%2freact-workspaces - Not found
```

**Comment by flashd2n** [comment]
Hello @mingfang 

There are a couple of typos in this readme. The correct name of the package is _@glue42/workspaces-ui-react_: 

`npm install @glue42/workspaces-ui-react`

Thank you for pointing this out, the readme has been updated and the changes will be published with #187

**Comment by mingfang** [comment]
I also had to
```
yarn add @glue42/react-hooks
```
